From 6bb3eb721ebea2046c2a1147f33e1a424ece038a Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Fri, 12 Mar 2021 15:06:36 +0100
Subject: [PATCH] remove BUILD_TESTING conditions

---
 CMakeLists.txt | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cd0c0dd..280920a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -34,11 +34,6 @@ PROJECT(${PROJECT_NAME} ${PROJECT_ARGS})
 # Project dependencies
 ADD_PROJECT_DEPENDENCY(sot-tools REQUIRED)
 
-IF(BUILD_TESTING)
-  FIND_PACKAGE(Boost REQUIRED COMPONENTS unit_test_framework)
-  ADD_PROJECT_DEPENDENCY(example-robot-data 3.8.0 REQUIRED)
-ENDIF(BUILD_TESTING)
-
 IF(BUILD_PYTHON_INTERFACE)
   FINDPYTHON()
   SEARCH_FOR_BOOST_PYTHON(REQUIRED)
@@ -84,6 +79,3 @@ IF(NOT INSTALL_PYTHON_INTERFACE_ONLY)
 ENDIF(NOT INSTALL_PYTHON_INTERFACE_ONLY)
 
 ADD_SUBDIRECTORY(src)
-IF(BUILD_TESTING)
-    ADD_SUBDIRECTORY(tests)
-ENDIF(BUILD_TESTING)
-- 
2.25.1

