From 8c7d4b27abff084344c8c6d05646ca64a6bf6d76 Mon Sep 17 00:00:00 2001
From: Guilhem Saurel <guilhem.saurel@laas.fr>
Date: Wed, 5 Oct 2022 12:41:53 +0200
Subject: [PATCH 2/2] CMake: force python 3 for noetic

---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8ea9615..36e2216 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -32,6 +32,7 @@ project(${PROJECT_NAME} ${PROJECT_ARGS})
 
 # Project dependencies
 if(BUILD_PYTHON_INTERFACE)
+  set(PYTHON_EXECUTABLE "/usr/bin/python3")
   add_project_dependency(dynamic-graph-python 4.0.0 REQUIRED)
   string(REGEX REPLACE "-" "_" PYTHON_DIR ${CUSTOM_HEADER_DIR})
 endif(BUILD_PYTHON_INTERFACE)
-- 
2.25.1

